src/status_im/translations/en.cljs

```diff
@@ -14,7 +14,7 @@
    :open                                  "Open"
    :description                           "Description"
    :enter-url                             "Enter URL"
-   :open-url                              "Open URL"
+   :open-dapp                             "Open ÐApp"
```

Comment:
this is new one, we don't have translations for it, so i renamed it

Reply:
What about other translations?

Reply:
what do you mean?

Reply:
Do we have to update the key in other translation files?

Reply:
Yes @jeluard @flexsurfer we do. Prob best to actually define a new checkpoint around here: https://github.com/status-im/status-react/blob/develop/src/status_im/i18n.cljs#L231

Reply:
so @jeluard @andytudhope how does this work, who should add this checkpoints?

Reply:
@flexsurfer Was `open-url` a new key that is not used anywhere? Is that case my bad it looks ok.

Reply:
The thinking at the time of writing that file was that whoever added/removed significant functionality would add a checkpoint, otherwise we would do it just for releases with major changes. 
As you are the one who has done a lot of these changes here, perhaps you could quickly add a new checkpoint (which inherits from the last, so it's really just documenting changes you have made) with any new key words. Not quite sure how to delete from last checkpoint if I'm honest... 😛

Reply:
@flexsurfer Are both `open-url` and `open-dapp` new keys?

Reply:
it was `open-url` new one in the previous PR, and i renamed it in this PR, but it's not only open-dapp i have more in this PR

Reply:
:+1: